topic02ex1/src/eventEmitter.js

```diff
@@ -0,0 +1,37 @@
+//export class EventEmitter...
+
+module.exports = class EventEmitter{
+    constructor(){
+        this.events = {};
+    }
+    on(eventName,callback){
+        if (typeof this.events[eventName] !== 'object') {
```

Comment:
this.events[eventName] =  this.events[eventName] || []
 this.events[eventName].push(callback)